7-layouts/css/styles.css

```diff
@@ -0,0 +1,160 @@
+.img-container {
+    flex-wrap: wrap;
+}
+
+header {
+    background-color: #323233;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+    color: white;
+    font-size: 25px;
+    justify-content: start;
+}
+
+.logo-fire {
+    width: 25px;
+    margin-right: 10px;
+}
+
+.banner-container {
+    width: 100%;
+    height: 35%;
+    position: relative;
+    box-sizing: border-box;
+    background-image: url('https://c.wallhere.com/photos/bf/60/cat_blue_eyes_face_animals_eyes-216035.jpg!d');
```

Comment:
if you check the layout example, these are content images and not decorative ones, please use `<img>` tags so they're accessible to screen readers.